api/agent/agent_test.go

```diff
@@ -109,8 +109,8 @@ func TestCallConfigurationRequest(t *testing.T) {
 	if model.ID == "" {
 		t.Fatal("model does not have id, GetCall should assign id")
 	}
-	if model.AppName != appName {
-		t.Fatal("app name mismatch", model.AppName, appName)
+	if model.AppID != app.ID {
```

Comment:
we need to check both

Reply:
I'm not sure that it's possible since call object no longer has AppName, it was substituted by AppID.

Reply:
Can call just have the *App object on it so it has both?

Reply:
think we all agreed on just using ids in the backend (call included) ? i abandon my original comment*